art_show/site_sections/art_show_applications.py

```diff
@@ -43,13 +43,6 @@ def index(self, session, message='', **params):
                 app.attendee = attendee
 
                 session.add(app)
-                send_email(
-                    c.ART_SHOW_EMAIL,
-                    app.email,
-                    'Art Show Application Received',
-                    render('emails/art_show/application.html',
-                           {'app': app}, encoding=None), 'html',
-                    model=app)
                 send_email(
```

Comment:
re: send_email still being in the code.

Reply:
This was converted into the new email fixture you see added above. We can't send them their agent code here because it doesn't exist yet.

Reply:
The other send_email function goes to the art show admin's email. It's not possible to convert that one to an automated email since we send the same type of email multiple times.

Reply:
Interesting!